integration-tests/src/tests.rs

```diff
@@ -115,6 +115,39 @@ fn test_take_i32() {
     run_test(cxx, hdr, rs, &["take_int"], &[]);
 }
 
+#[test]
+fn test_nested_module() {
```

Comment:
What's the behavior if there's a `#[cxx::bridge]` mod nested within another mod? I seem to remember that such a mod is ignored by cxx's standard code gen tool, so we might be going a little further here with autocxx_gen. Could you check?

I'd prefer not to go further than `cxx` does for the case of `cxx::bridge` mods, so perhaps we could throw a parse error if we find such a mod nested inside another mod.

I may be totally wrong - it might just work already with plain `cxx`.

Reply:
Seems to me like it works. I tried it at e2b7114d02478fdc9db21588bb164c9d6e947d37.

Let me know if you think that'd be helpful to merge, it doesn't actually have anything to do with autocxx so this doesn't seem like the right place for it.

Reply:
Nope, that's fine, thanks very much for trying it out.